pkg/services/ngalert/accesscontrol.go

```diff
@@ -125,16 +125,10 @@ var (
 			DisplayName: "Notifications Editor",
 			Description: "Can add, update, and delete contact points and notification policies in Grafana and external providers",
 			Group:       AlertRolesGroup,
-			Version:     1,
+			Version:     2,
```

Comment:
You'd also need to update the `alertingWriterRole` version. I'm not a big fan of the `ConcatPermissions` function for this specific problem.

Reply:
Done. Thanks for pointing out